common/djangoapps/third_party_auth/tests/test_admin.py

```diff
@@ -0,0 +1,71 @@
+"""
+Tests third_party_auth admin views
+"""
+import unittest
+
+from django.conf import settings
+from django.contrib.admin.sites import AdminSite
+from django.core.urlresolvers import reverse
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.forms import models
+
+from student.tests.factories import UserFactory
+from third_party_auth.admin import OAuth2ProviderConfigAdmin
+from third_party_auth.models import OAuth2ProviderConfig
+from third_party_auth.tests import testutil
+
+
+# This is necessary because cms does not implement third party auth
+@unittest.skipUnless(settings.FEATURES.get('ENABLE_THIRD_PARTY_AUTH'), 'third party auth not enabled')
+class Oauth2ProviderConfigAdminTest(testutil.TestCase):
+    """
+    Tests for oauth2 provider config admin
+    """
+    def test_oauth2_provider_edit_icon_image(self):
```

Comment:
Please add a docstring that clearly explains why this test is important, given our use of a KeyedConfigurationModel with a FileField.